components/export/api/src/main/java/org/phenotips/export/internal/DataToCellConverter.java

```diff
@@ -500,6 +500,16 @@ public DataSection genesBody(Patient patient) throws Exception
             y++;
         }
 
+        /* Notes export */
+        if (present.contains("gene_comments")) {
```

Comment:
I think this will never be true, this value is never placed in the `present` list. Perhaps use `genes_comments` here, while still using `gene_comments` two lines down?